spec/corrector_spec.rb

```diff
@@ -0,0 +1,23 @@
+# require './capitalize_decorator'
```

Comment:
- [ ] Kindly un-comment this file and the other commented file as well and make sure all tests are passing